tests/Listeners/ClearInstancesListenerTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace AvtoDev\RoadRunnerLaravel\Tests\Listeners;
+
+use Mockery as m;
+use AvtoDev\RoadRunnerLaravel\Events\Contracts\WithApplication;
+use AvtoDev\RoadRunnerLaravel\Listeners\ClearInstancesListener;
+use Illuminate\Contracts\Config\Repository as ConfigRepository;
+
+/**
+ * @covers \AvtoDev\RoadRunnerLaravel\Listeners\ClearInstancesListener<extended>
+ */
+class ClearInstancesListenerTest extends AbstractListenerTestCase
```

Comment:
Add singleton binding check

Reply:
done in ada5924a4d3bed7e4b26778a6390842cf5872a06